toolkits/global/packages/global-lists/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@springernature/global-lists",
+  "version": "1.0.0",
+  "license": "MIT",
+  "description": "Provide different styled lists",
+  "keywords": [],
+  "author": "Springer Nature",
+  "brandContext": "^6.0.0"
```

Comment:
Current version is `8.0.1`, is there a reason for using `6.0.0`?

Reply:
Maybe copy paste from `springer-lists`, I can fix that

Reply:
Thinking about this again, I think we should leave this, so that Springer has a version tied to the brand that they are using. Then can release another version using the latest

Reply:
It depends if you're deleting the springer-lists package. If not, then our usage of that will keep us tied to that 6.0.0 brand context.

Reply:
Then we can check any issues with swapping to the global one and delete the springer-lists if all is good?